test/javascripts/unit/lib/chat-window-store-test.js

```diff
@@ -4,6 +4,7 @@ import { test } from "qunit";
 discourseModule("Discourse Chat | Unit | chat-window-store", function (hooks) {
   hooks.beforeEach(function () {
     this.chatWindowStore = this.container.lookup("service:chat-window-store");
+    this.chatWindowStore.set("fullPage", false);
```

Comment:
Usually, we want to avoid cleaning up state in a `beforeEach` callback since it does not address the problem where certain tests are leaking state. Will moving this to the `afterEach` callback resolve the flakiness seen here?

Reply:
Ah okay. Let's try and see.

Reply:
Ah okay. Let's try and see.